test/e2e/placement_negative_cases_test.go

```diff
@@ -342,4 +343,187 @@ var _ = Describe("handling errors and failures gracefully", func() {
+						ClusterNames: []string{
+							memberCluster1EastProdName,
+						},
+					},
+					Strategy: placementv1beta1.RolloutStrategy{
+						Type: placementv1beta1.RollingUpdateRolloutStrategyType,
+						RollingUpdate: &placementv1beta1.RollingUpdateConfig{
+							UnavailablePeriodSeconds: ptr.To(2),
+						},
+					},
+				},
+			}
+			Expect(hubClient.Create(ctx, crp)).To(Succeed(), "Failed to create CRP")
+		})
+
+		It("should update CRP status as expected", func() {
+			crpStatusUpdatedActual := func() error {
+				crp := &placementv1beta1.ClusterResourcePlacement{}
+				if err := hubClient.Get(ctx, types.NamespacedName{Name: crpName}, crp); err != nil {
+					return err
+				}
+
+				wantStatus := placementv1beta1.PlacementStatus{
+					Conditions: crpAppliedFailedConditions(crp.Generation),
+					PerClusterPlacementStatuses: []placementv1beta1.PerClusterPlacementStatus{
+						{
+							ClusterName:           memberCluster1EastProdName,
+							ObservedResourceIndex: "0",
+							FailedPlacements: []placementv1beta1.FailedResourcePlacement{
+								{
+									ResourceIdentifier: placementv1beta1.ResourceIdentifier{
+										Group:     "",
+										Version:   "v1",
+										Kind:      "ConfigMap",
+										Namespace: workNamespaceName,
+										Name:      wrappedCMName1,
+										Envelope: &placementv1beta1.EnvelopeIdentifier{
+											Name:      envelopeName,
+											Namespace: workNamespaceName,
+											Type:      placementv1beta1.ResourceEnvelopeType,
+										},
+									},
+									Condition: metav1.Condition{
+										Type:               placementv1beta1.WorkConditionTypeApplied,
+										Status:             metav1.ConditionFalse,
+										Reason:             string(workapplier.ApplyOrReportDiffResTypeDuplicated),
+										ObservedGeneration: 0,
+									},
+								},
+							},
+							Conditions: perClusterApplyFailedConditions(crp.Generation),
+						},
+					},
+					SelectedResources: []placementv1beta1.ResourceIdentifier{
+						{
+							Kind:    "Namespace",
+							Name:    workNamespaceName,
+							Version: "v1",
+						},
+						{
+							Group:     placementv1beta1.GroupVersion.Group,
+							Kind:      placementv1beta1.ResourceEnvelopeKind,
+							Version:   placementv1beta1.GroupVersion.Version,
+							Name:      envelopeName,
+							Namespace: workNamespaceName,
+						},
+					},
+					ObservedResourceIndex: "0",
+				}
+				if diff := cmp.Diff(crp.Status, wantStatus, placementStatusCmpOptions...); diff != "" {
+					return fmt.Errorf("CRP status diff (-got, +want): %s", diff)
+				}
+				return nil
+			}
+			Eventually(crpStatusUpdatedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to update CRP status as expected")
+			Consistently(crpStatusUpdatedActual, consistentlyDuration, consistentlyInterval).Should(Succeed(), "CRP status has changed unexpectedly")
+		})
+
+		It("should place some manifests on member clusters", func() {
```

Comment:
will we place the remaining valid manifests? or just abort in the middle?

Reply:
Yeah, for the duplication case, all valid manifests will be applied (e.g, with manifests A, A, B, C, D, D -> the applier will apply A, B, C, D).